frontend/src/components/teacher/student-management/classroom-summary/ClassroomPopover.tsx

```diff
@@ -24,6 +25,7 @@ const ClassroomPopover = ({
     onClose: onPopoverClose,
   } = useDisclosure();
   const [showDeleteModal, setShowDeleteModal] = React.useState(false);
+  const [showEditModal, setShowEditModal] = React.useState(false);
```

Comment:
`useDisclosure` from `@chakra-ui/react` would be the canonical way to do it, but I'll leave it up to you since I think we've used both patterns in the codebase

Reply:
ok sounds good! WIll keep this in mind for future changes thank you